autoPyTorch/optimizer/smbo.py

```diff
@@ -205,6 +213,24 @@ def __init__(self,
                                               port=self.logger_port)
         self.logger.info("initialised {}".format(self.__class__.__name__))
 
+        # read and validate initial configurations
+        initial_configurations = json.load(open(os.path.join(os.path.dirname(__file__),
+                                                             '../configs/greedy_portfolio.json')))
+
+        self.initial_configurations: typing.Optional[typing.List[Configuration]] = None
+        assert portfolio_selection in ['none', 'greedy']
```

Comment:
Can you make this a value error so users can catch it if needed? And probably it makes sense to do so in the API so that we don't even proceed with an incorrect option.

Also, how can the user provide a greedy portfolio json file? Can this be a file provided by the user?

Reply:
yeah that also makes sense. I'll add this